src/index.js

```diff
@@ -1,12 +1,32 @@
 import React from 'react'
 import { render } from 'react-dom'
-import { createStore } from 'redux'
+import { createStore, applyMiddleware } from 'redux'
 import { Provider } from 'react-redux'
 import App from './containers/App'
 import reducer from './reducers'
 import 'todomvc-app-css/index.css'
 
-const store = createStore(reducer)
+import { createCaptureMiddleware, createPersistHandler, createRetrieveHandler, createRetrieveMiddleware, createReplayMiddleware, wrapReducer } from 'redux-vcr';
+
+const firebaseAuth = {
+  apiKey: "AIzaSyBklunZBwf0YMFbpHWVmfcu2scFAb2gq3o",
+  authDomain: "todomvc-ccb8e.firebaseapp.com",
+  databaseURL: "https://todomvc-ccb8e.firebaseio.com",
+};
+
+const persistHandler = createPersistHandler({ firebaseAuth });
+const retrieveHandler = createRetrieveHandler({ firebaseAuth });
+
+const middlewares = [
+  createCaptureMiddleware({ persistHandler }),
+  createRetrieveMiddleware({ retrieveHandler }),
+  createReplayMiddleware({ maximumDelay: 1000 }),
+];
```

Comment:
Those handlers are passed to their respective middlewares to allow pushing and pulling when appropriate actions are dispatched.

Additionally, the `replayMiddleware` handles playing cassettes.